src/FM.LiveSwitch.Mux/MuxOptions.cs

```diff
@@ -6,10 +6,10 @@ namespace FM.LiveSwitch.Mux
 {
     public class MuxOptions
     {
-        public const int MinMargin = 0;
-        public const int MinWidth = 160;
-        public const int MinHeight = 120;
-        public const char ArgSeparator = ',';
+        public static readonly int MinMargin = 0;
```

Comment:
Codacy found an issue: [Remove this initialization to 'MinMargin', the compiler will do that for you.](https://app.codacy.com/gh/liveswitch/liveswitch-mux/pullRequest?prid=6354353)